app/controllers/albums.js

```diff
@@ -0,0 +1,19 @@
+const albumService = require('../services/albums');
+
+exports.getAlbums = async (req, res, next) => {
+  try {
+    const albumsRequest = await albumService.getAlbums();
```

Comment:
`albums` instead of `albumsRequest` seems good, because yo are getting the albums